packages/viewer/src/store/modules/ui/actions/addErrors.ts

```diff
@@ -0,0 +1,30 @@
+import log, { LogPreDefinedColor } from '@swissgeo/log'
+import { ErrorMessage } from '@swissgeo/log/Message'
+
+import type { UIStore } from '@/store/modules/ui/types/ui'
+import type { ActionDispatcher } from '@/store/types'
+
+export default function addErrors(
+    this: UIStore,
+    errors: ErrorMessage[],
+    dispatcher: ActionDispatcher
+): void {
+    if (Array.isArray(errors) && errors.every((error) => error instanceof ErrorMessage)) {
+        errors
+            .filter(
+                (error) =>
+                    ![...this.errors].some((otherError) => error.isEqual(otherError))
+            )
+            .forEach((error) => {
+                this.errors.add(error)
+            })
+    } else {
+        log.error({
+            title: 'UI store / addErrors',
+            titleStyle: {
+                color: LogPreDefinedColor.Red,
+            },
```

Comment:
That's an "old" way of styling the log, which is now invalid, can you switch to
```TS
titleColor: LogPreDefinedColor.Red,
```
when you see this notation?